dialect/sql/builder.go

```diff
@@ -968,6 +968,19 @@ func (p *Predicate) EQ(col string, arg interface{}) *Predicate {
 	})
 }
 
+func ColEQ(col1 string, col2 string) *Predicate {
```

Comment:
What do you think about naming these predicates as `Columns<Op>` (e.g. `ColumnsEQ`) instead of `Col<Op>`? It's a little bit longer, but more clear IMO. 

BTW, I just want to mention that this query-builder API is going to be changed soon into a new implementation (I don't like this one at all).